NitroxPatcher/Patches/Dynamic/ResourceTracker_OnPickedUp_Patch.cs

```diff
@@ -0,0 +1,27 @@
+using System.Reflection;
+using NitroxClient.GameLogic;
+using NitroxModel.Core;
+using NitroxModel.Helper;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+/// <summary>
+/// Event when someone picks up an item.
+/// </summary>
+public sealed partial class ResourceTracker_OnPickedUp_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((ResourceTracker d) => d.OnPickedUp(default));
+    
+    public static bool Prefix(ResourceTracker __instance, Pickupable p)
```

Comment:
I don't think you need pickupable here as you don't make use of it
```suggestion
    public static bool Prefix(ResourceTracker __instance)
```